File: Java/src/main/java/com/DSA/Java/LinkedList/CircularLinkedList/Runner.java
package com.DSA.Java.LinkedList.CircularLinkedList;

public class Runner {
   public static void main(String[] args) {
      CircularSinglyLinkedList cList = new CircularSinglyLinkedList();
      cList.createCircularLinkedList(5);

      cList.insertElement(4, 0);
      cList.insertElement(6, 1);
      cList.insertElement(7, 8);

      cList.traverseList();

      // cList.deleteList();
      // cList.traverseList();
      
      // cList.deleteElement(0);
      // cList.deleteElement(2);
      // cList.traverseList();

      // System.out.println(cList.searchElement(20));
      // System.out.println(cList.searchElement(25));
      // System.out.println(cList.searchElement(40));
   }
}

File: C++/LinkedList/Single/Circular/Runner.cpp
#include <iostream>
#include "SinglyCircularLL.h"

int main() {
   SinglyCircularLL sll;
   sll.insert_element(5, 0);
   sll.insert_element(15, 1);
   sll.insert_element(20, 2);
   sll.insert_element(10, 1);
   sll.insert_element(25, 5);
   sll.traverse();

   sll.delete_element(5);
   sll.traverse();

   sll.delete_element(20);
   sll.traverse();

   sll.delete_element(35);
   sll.traverse();

   sll.delete_list();
   sll.traverse();

   return 0;
}
